docs/pages/versions/unversioned/guides/contributing-to-web.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Contributing to Web
+---
+
+There are a couple different things you could work on in regards to Expo for web. Here we'll cover how you can setup your environment to easily contribute.
+
+## 💎 Features & Tests
+
+To work on modules like `expo-camera` or even community libraries, I like to work in the expo monorepo, this is because it's a yarn workspace which links the packages really well.
```

Comment:
To work on modules like `expo-camera` or community libraries, consider working in the expo monorepo. The reason for this is that the monorepo is a yarn workspace which links the packages really well.